### DISCUSSION TITLE
Facing issues while deploying the JavaScript Portfolio website using Azure

### DISCUSSION THREAD

**Discussion by kartikver15gr8** [microsoft/SATechnicalOnboarding#37] [Questions]
I completed the JavaScript website and made the adequate changes as well, also hosted it using GitHub pages, but when I'm trying to host it using azure, its popping me up with a prompt that says ```"Select Subscriptions"``` and I'm not been able to select any subscriptions.

What should I do? 
can someone help

It shows me this 

![image](https://github.com/microsoft/SATechnicalOnboarding/assets/91731654/f3e4a130-ebc3-47c9-93a0-f6340f7b9c4d)

**Comment by dileepadev-old** [comment]
Hello @kartikver15gr8, If you are using Visual Studio Code (VS Code), try to log into your Azure account (it is recommended to log in with your Microsoft Learn Student Ambassador Azure account) in VS Code. Then you will be able to see your student's free subscription. If you haven't activated your Azure subscription, please check out the guideline; it will clearly explain how to activate your account. I think this way will solve your issue, right? 😊🚩

**Reply by kajallochab** [reply]
I have activated my Azure subscription, but I am still getting the same issue as mentioned above by @kartikver15gr8

**Reply by AnthonyByansi** [reply]
Hello @kajallochab, I suggest that you install the [Azure Account VS Code Extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode.azure-account) and proceed to sign in using your Microsoft Learn Student Ambassador (MLSA) account. Thank you.
